StandaloneViewer/StandaloneViewer/client/components/toolbarSection/toolbarSection.js

```diff
@@ -68,7 +68,7 @@ Template.toolbarSection.helpers({
         });
 
         extraTools.push({
-            id: 'dragProbe',
+            id: 'probe',
```

Comment:
We intend to keep dragProbe, so switch this back when that tool has been migrated.